src/components/file/file.tsx

```diff
@@ -164,25 +175,48 @@ export class File {
                 disabled={this.disabled || this.readonly}
             />,
             <limel-chip-set
+                class={{
+                    'is-file-picker': true,
+                    'shows-dropzone': true,
+                    'highlight-dropzone': this.isDraggingOverDropZone,
```

Comment:
I added some interactivity to the styles 🙂

https://user-images.githubusercontent.com/154725/126768512-9a88a5d2-71c3-4693-8625-36a1b74337ae.mov

Reply:
There may be more cool things we can do with changing the pointer and stuff, but I don't have time before my vacation 😅

Reply:
Lovely, so you basically wanna close this issue: https://github.com/Lundalogik/crm-feature/issues/2249 😄 in the same time

Reply:
Wh have [an issue](https://github.com/Lundalogik/crm-feature/issues/2249) for this. It can wait for perfection after the vacation ;)

Reply:
Ah, yeah, I guess it would be really nice to have _all_ drop zones light up a bit when the user drags something anywhere on the page, and then still have each individual drop zone become even more highlighted when the user drags over it.